src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -1230,6 +1230,16 @@ object LazyList extends SeqFactory[LazyList] {
     at(0)
   }
 
+  override def range[A: Integral](start: A, end: A): LazyList[A] =
+    range(start, end, step = Integral[A].one)
```

Comment:
```suggestion
    newLL(rangeImpl(start, end, step = Integral[A].one))
```

this is important to ensure `Integral[A].one` is evaluated lazily, in case for some reason that matters

Reply:
Oh right, I thought that was part of the previous commit suggestion and just forgot to check 😅 
Times Luis did something dumb out of laziness this week: 2